app/scripts/store/dimStoreItems.directive.js

```diff
@@ -80,6 +80,22 @@
         '        </div>',
         '      </div>',
         '    </div>',
+        '    <div ng-if="::vm.store.id !== \'vault\'" class="title">',
+        '      <span>Vendors</span>',
+        '    </div>',
+        '    <div class="sub-section sort-progression">',
+        '      <div class="unequipped">',
+        '        <span class="item" ng-if="faction.color" ng-repeat="faction in vm.store.progression.progressions track by $index" title="{{faction.identifier}}\n{{faction.progressToNextLevel}}/{{faction.nextLevelAt}}">',
+        '          <svg width="48" height="48">',
+        '            <polygon stroke-dasharray="130" fill="{{faction.color}}" points="24,1 47,24 24,47 1,24"/>',
+        '            <image xlink:href="" ng-attr-xlink:href="{{faction.icon | bungieIcon}}" ng-attr-x="{{faction.scale === \'.8\' ? 6 : 48-(faction.scale*48)}}" ng-attr-y="{{faction.scale === \'.8\' ? 6 : 48-(faction.scale*48)}}" width="48" height="48" ng-attr-transform="scale({{faction.scale}})" />',
+        '            <polygon stroke-dasharray="130" fill-opacity="0" stroke="#FFF" stroke-opacity=".35" stroke-width="2" points="24,1 47,24 24,47 1,24"/>',
+        '            <polygon stroke-dasharray="130" style="stroke-dashoffset:{{130-(130*faction.progressToNextLevel/faction.nextLevelAt)}}" fill-opacity="0" stroke="#FFF" stroke-width="2" points="24,1 47,24 24,47 1,24"/>',
+        '          </svg>',
```

Comment:
don't look too closely... it works but it's... scary >_<

would like the stroke ends of the polygon path to properly intersect...

<img width="66" alt="screen shot 2016-05-16 at 1 12 59 am" src="https://cloud.githubusercontent.com/assets/424158/15280895/bc96a52e-1b03-11e6-82cd-0c63283a5a96.png">